.gitattributes

```diff
@@ -0,0 +1,32 @@
+/.babelrc export-ignore
+/.editorconfig export-ignore
+/.env export-ignore
+/.eslintrc.js export-ignore
+/.git export-ignore
+/.gitattributes export-ignore
+/.github export-ignore
+/.gitignore export-ignore
+/.phpspec.yml export-ignore
+/.prettierrc export-ignore
+/.stylelintrc export-ignore
+/.travis.yml export-ignore
+/behat.yml export-ignore
+/docker-compose.yml export-ignore
+/ecs.php export-ignore
+/Makefile export-ignore
+/package.json export-ignore
+/package-lock.json export-ignore
+/phpstan.neon export-ignore
+/phpunit.bootstrap.php export-ignore
+/phpunit.xml.dist export-ignore
+/postcss.config.js export-ignore
+/run_behat_tests.sh export-ignore
+/symfony.lock export-ignore
+/webpack.config.js export-ignore
+/assets export-ignore
+/bin export-ignore
```

Comment:
If we exclude this, won't we exclude `bin/console` as well? Or is that no problem because `bolt/project` has its own copy?

Reply:
For me "bolt/core" should be taken as a Symfony component, we just need to have the Bolt business code for everything to work properly.

"bolt/project" is a symfony project, so for any symfony project the "bin/console" file must be versioned.

I think this is the best solution to make the deposit lighter while waiting to clean up.

Reply:
That makes sense! Let's roll with this! 👍